<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTreeWidget" name="navTree">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="BookTableView" name="bookTable">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>24</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>20</number>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddBooks"/>
    <addaction name="actionRemoveBooks"/>
    <addaction name="actionEditMetadata"/>
    <addaction name="separator"/>
    <addaction name="actionConvertToDisk"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewLog"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="ToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAddBooks">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-add.png</normaloff>:/toolbar/tool-add.png</iconset>
   </property>
   <property name="text">
    <string>Add books</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionConvertToDisk">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-folder.png</normaloff>:/toolbar/tool-folder.png</iconset>
   </property>
   <property name="text">
    <string>Convert to folder</string>
   </property>
   <property name="toolTip">
    <string>Convert book to folder</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-settings.png</normaloff>:/toolbar/tool-settings.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Libro preferences</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionRemoveBooks">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-delete.png</normaloff>:/toolbar/tool-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove books</string>
   </property>
   <property name="toolTip">
    <string>Remove books</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionEditMetadata">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-edit.png</normaloff>:/toolbar/tool-edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit metadata</string>
   </property>
   <property name="toolTip">
    <string>Edit metadata</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionViewLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionSendToReader">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-reader.png</normaloff>:/toolbar/tool-reader.png</iconset>
   </property>
   <property name="text">
    <string>Send to reader</string>
   </property>
   <property name="toolTip">
    <string>Send selected books to reader device</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSendBooksViaMail">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/tool-mail.png</normaloff>:/toolbar/tool-mail.png</iconset>
   </property>
   <property name="text">
    <string>Send books via mail</string>
   </property>
   <property name="toolTip">
    <string>Send books via mail</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BookTableView</class>
   <extends>QTableView</extends>
   <header>.booktableview.h</header>
  </customwidget>
  <customwidget>
   <class>ToolBar</class>
   <extends>QToolBar</extends>
   <header>.toolbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionAddBooks</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionAddBooks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConvertToDisk</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionConvertToDisk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemoveBooks</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionRemoveBooks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutQt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionAboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditMetadata</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionEditMetadata()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionViewLog</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionViewLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onActionAddBooks()</slot>
  <slot>onActionConvertToDisk()</slot>
  <slot>onActionSettings()</slot>
  <slot>onActionRemoveBooks()</slot>
  <slot>onActionAbout()</slot>
  <slot>onActionAboutQt()</slot>
  <slot>onActionEditMetadata()</slot>
  <slot>onActionViewLog()</slot>
 </slots>
</ui>
